cypress/e2e/pages/xeniumProbeHybridisationSpec.cy.ts

```diff
@@ -266,7 +266,7 @@ describe('Xenium Probe Hybridisation', () => {
       });
 
       it('shows a success message', () => {
-        cy.findByText('Xenium probe hybridisation recorded on all labware').should('be.visible');
+        cy.findByText('Xenium probe hubridisation recorded on all labware').should('be.visible');
```

Comment:
Something gone wring with merge it seems. this typo was fixed in the other branch which is again reverted back now. Please change to 'hybridisation'